BuiltProjectOutputGroupDependencies target missing after updating Nerdbank.GitVersioning to 3.5.x

Hey!

After upgrading to the latest 3.5.103 a VSIX project I have it's failing with the error:

`C:\Users\VssAdministrator\.nuget\packages\nerdbank.gitversioning\3.5.103\build\PrivateP2PCaching.proj : error MSB4057: The target "BuiltProjectOutputGroupDependencies" does not exist in the project.`

Reverting to 3.4.255 "fixes" it. Any hints on where I should look?

--- a/src/Nerdbank.GitVersioning.Tasks/build/PrivateP2PCaching.proj
+++ b/src/Nerdbank.GitVersioning.Tasks/build/PrivateP2PCaching.proj
@@ -20,6 +20,7 @@
   <!-- These targets are called when the consuming project is a VSIX project. -->
   <Target Name="BuiltProjectOutputGroup" />
   <Target Name="SatelliteDllsProjectOutputGroup" />
+  <Target Name="BuiltProjectOutputGroupDependencies" />
 
   <Import Project="$(NBGV_PrivateP2PAuxTargets)" Condition=" '$(NBGV_PrivateP2PAuxTargets)' != '' " />
 </Project>

--- a/src/Nerdbank.GitVersioning.Tasks/build/PrivateP2PCaching.proj
+++ b/src/Nerdbank.GitVersioning.Tasks/build/PrivateP2PCaching.proj
@@ -21,6 +21,7 @@
   <Target Name="BuiltProjectOutputGroup" />
   <Target Name="SatelliteDllsProjectOutputGroup" />
   <Target Name="BuiltProjectOutputGroupDependencies" />
+  <Target Name="DebugSymbolsProjectOutputGroup" />
 
   <Import Project="$(NBGV_PrivateP2PAuxTargets)" Condition=" '$(NBGV_PrivateP2PAuxTargets)' != '' " />
 </Project>

